src/Jackett.Common/Indexers/Xthor.cs

```diff
@@ -204,74 +211,85 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
-                        //TODO replace with download link?
-                        var guid = new Uri(TorrentDescriptionUrl.Replace("{id}", torrent.id.ToString()));
-                        var comments = new Uri(TorrentCommentUrl.Replace("{id}", torrent.id.ToString()));
-                        var link = new Uri(torrent.download_link);
-                        var release = new ReleaseInfo
-                        {
-                            // Mapping data
-                            Category = MapTrackerCatToNewznab(torrent.category.ToString()),
-                            Title = torrent.name,
-                            Seeders = torrent.seeders,
-                            Peers = torrent.seeders + torrent.leechers,
-                            MinimumRatio = 1,
-                            MinimumSeedTime = 345600,
-                            PublishDate = publishDate,
-                            Size = torrent.size,
-                            Grabs = torrent.times_completed,
-                            Files = torrent.numfiles,
-                            UploadVolumeFactor = 1,
-                            DownloadVolumeFactor = (torrent.freeleech == 1 ? 0 : 1),
-                            Guid = guid,
-                            Comments = comments,
-                            Link = link,
-                            TMDb = torrent.tmdb_id
-                        };
-
-                        //TODO make consistent with other trackers
-                        if (DevMode)
+                    // If contains torrents
+                    if (xthorResponse.torrents != null)
+                    {
+                        // Adding each torrent row to releases
+                        releases.AddRange(xthorResponse.torrents.Select(torrent =>
                         {
-                            Output(release.ToString());
-                        }
+                            //issue #3847 replace multi keyword
+                            if (!string.IsNullOrEmpty(ReplaceMulti))
+                            {
+                                var regex = new Regex("(?i)([\\.\\- ])MULTI([\\.\\- ])");
+                                torrent.name = regex.Replace(torrent.name, "$1" + ReplaceMulti + "$2");
+                            }
+
+                            // issue #8759 replace vostfr and subfrench with English
+                            if (ConfigData.Vostfr.Value)
+                                torrent.name = torrent.name.Replace("VOSTFR", "ENGLISH").Replace("SUBFRENCH", "ENGLISH");
+
+                            var publishDate = DateTimeUtil.UnixTimestampToDateTime(torrent.added);
+                            //TODO replace with download link?
+                            var guid = new Uri(TorrentDescriptionUrl.Replace("{id}", torrent.id.ToString()));
+                            var comments = new Uri(TorrentCommentUrl.Replace("{id}", torrent.id.ToString()));
```

Comment:
Change this code for me:
```
        private string TorrentCommentUrl => TorrentDescriptionUrl;
        private string TorrentDescriptionUrl => SiteLink + "details.php?id={id}";
```
to
```
private string TorrentCommentUrl => SiteLink + "details.php?id=";
```

and
```
                            //TODO replace with download link?
                            var guid = new Uri(TorrentDescriptionUrl.Replace("{id}", torrent.id.ToString()));
                            var comments = new Uri(TorrentCommentUrl.Replace("{id}", torrent.id.ToString()));
```
to
```
var comments = new Uri(TorrentCommentUrl + id);
```

and

```
 Guid = guid,
 Comments = comments,
```
to
```
 Guid = comments,
 Comments = comments,
```

remember to test the details link is working after this change